solidity-v1/dashboard/src/components/modal/staking/claim/TokensClaimed.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { ModalHeader, ModalBody, ModalFooter } from "../../Modal"
+import { ViewInBlockExplorer } from "../../../ViewInBlockExplorer"
+import * as Icons from "../../../Icons"
+import Button from "../../../Button"
+import { colors } from "../../../../constants/colors"
+import { withBaseModal } from "../../withBaseModal"
+
+const TokensClaimedComponent = ({ txHash, onClose }) => {
+  return (
+    <>
+      <ModalHeader>Claim</ModalHeader>
+      <ModalBody>
+        <h3 className="flex row center mb-1">
+          <Icons.OK color={colors.success} />
+          <span className="ml-1">Success!</span>
+        </h3>
+        <h4 className="text-grey-70 mb-2">
+          View your transaction&nbsp;
+          <ViewInBlockExplorer
+            type="tx"
+            className="text-grey-70"
+            id={txHash}
+            text="here"
+          />
```

Comment:
There is no arrow in FIgma next to `here` link. Can we add the posisbility to remove it in `ViewInBlockExplorer` component?

Reply:
I left a comment about this in [Figma](https://www.figma.com/file/Su61MwHLrQn3ld24q85pMx?node-id=11784:87075#124245039) cc @liz-shinn @sashatanase. This is an external link and for any other links we always display an arrow indicating that it is an external link.

Reply:
Yup! Will make that update in the Figma file. 

There were a few other things I tweaked as well - all of the modal button text and titles were default ones (I believe they were overridden at some point by changes to the master component in the DS file) so I corrected those to what they should have been.